src/qibolab/_core/instruments/qm/components/configs.py

```diff
@@ -22,6 +23,27 @@
 
 DEFAULT_SAMPLING_RATE = 1e9
 
+DEFAULT_FEEDFORWARD_MAX = 2 - 2**-16
+"""Maximum feedforward tap value"""
+DEFAULT_FEEDBACK_MAX = 1 - 2**-20
+"""Maximum feedback tap value"""
+
+
+def normalize_feedforward(taps: list[float], threshold: float):
+    max_value = np.max(np.abs(taps))
+    if max_value > threshold:
+        return (threshold * np.array(taps / max_value)).tolist()
+    else:
+        return taps
```

Comment:
This should be equivalent:
```suggestion
    scale = np.max(np.abs(taps) / threshold, initial=1)
    return (np.array(taps) / scale).tolist()
```

In the original function, a value of at most `1.` is multiplied by the original `taps` (only if `threshold <= max_value` you multiply by `threshold / max_value`).
This is just the same, dividing for a value that is at least `1.`.

To me, it's not only more concise, but rather more explicit.